jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -48,14 +50,16 @@ class SpectralExtraction(PluginTemplateMixin, ApertureSubsetSelectMixin,
       Subset to use for the spectral extraction, or ``Entire Cube``.
     * ``add_results`` (:class:`~jdaviz.core.template_mixin.AddResults`)
     * :meth:`collapse`
+    * ``aperture_method`` (:class:`~jdaviz.core.template_mixin.SelectPluginComponent`):
+      Method used to create a cone aperture in spectral extraction.
```

Comment:
This isn't only used for a cone though, maybe use the same text as in the UI hint (and we can run it by POs/@jenneh)